### PR TITLE
en,zh: add note for upgrade cmd after v1.3.0

### PR DESCRIPTION
**Pull request by KanShiori**
### What is changed, added, or deleted? (Required)
Close https://github.com/pingcap/tidb-operator/issues/4275

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: en/upgrade-tidb-operator.md [hunk 1]
```diff
@@ -82,7 +86,7 @@ If your server has access to the internet, you can perform online upgrade by tak
 
     > **Note:**
     >
-    > After TiDB Operator is upgraded, the `discovery` Deployment in all TiDB clusters is automatically upgraded to the corresponding version of TiDB Operator.
+    > If upgrading Operator to V1.3.0 or later for the first time, you can't use `replace` command to upgrade TidbNGMonitoring CRD due to TidbNGMonitoring CRD is new, you need to create TidbNGMonitoring first.
```
Comment (KanShiori) [inline en/upgrade-tidb-operator.md:89]: TODO: 英文等中文 Review 后再修改

File: zh/upgrade-tidb-operator.md [hunk 2]
```diff
@@ -21,24 +21,44 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/upgrade-tidb-operator/']
     如果输出中未包含你需要的新版本，可以使用 `helm repo update` 命令更新 repo。详情请参考[配置 Helm repo](tidb-toolkit.md#配置-helm-repo)。
 
 2. 更新 Kubernetes 的 CustomResourceDefinition (CRD)。关于 CRD 的更多信息，请参阅 [CustomResourceDefinition](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/)。
-
+   
     * 如果 Kubernetes 版本大于等于 1.16:
 
-        {{< copyable "shell-regular" >}}
+        1. 如果是第一次升级 Operator 到 v1.3.0 及以后版本，需要先创建新增加的 TidbNGMonitoring CRD。
```
Comment (DanielZhangQD) [inline zh/upgrade-tidb-operator.md:27]:
```suggestion
        1. 如果 TiDB Operator 从 v1.3.x 之前版本升级到 v1.3.x 及以后版本，需要先执行下面命令创建新增加的 TidbNGMonitoring CRD，否则，直接执行下一步。
```

File: zh/upgrade-tidb-operator.md [hunk 3]
```diff
@@ -21,24 +21,44 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/upgrade-tidb-operator/']
...
+
+            ```shell
+            kubectl replace -f https://raw.githubusercontent.com/pingcap/tidb-operator/${operator_version}/manifests/crd.yaml && \
+            kubectl get crd tidbclusters.pingcap.com
+            ```
 
     * 如果 Kubernetes 版本小于 1.16:
 
-        {{< copyable "shell-regular" >}}
+        1. 如果是第一次升级 Operator 到 v1.3.0 及以后版本，需要先创建新增加的 TidbNGMonitoring CRD。
```
Comment (DanielZhangQD) [inline zh/upgrade-tidb-operator.md:46]:
```suggestion
        1. 如果 TiDB Operator 从 v1.3.x 之前版本升级到 v1.3.x 及以后版本，需要先执行下面命令创建新增加的 TidbNGMonitoring CRD，否则，直接执行下一步。
```

File: zh/upgrade-tidb-operator.md [hunk 4]
```diff
@@ -132,15 +152,23 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/upgrade-tidb-operator/']
 
 2. 将下载的文件和镜像上传到需要升级的服务器上，在服务器上按照以下步骤进行安装：
 
-    1. 安装 TiDB Operator 需要的 `crd.yaml` 文件：
+    1. 如果是第一次升级 Operator 到 v1.3.0 及以后版本，需要先创建新增加的 TidbNGMonitoring CRD。
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        kubectl create -f ./crd.yaml
```
Comment (DanielZhangQD) [inline zh/upgrade-tidb-operator.md:160]: 这里要说一下其他 crd 会报错，可以忽略

File: zh/upgrade-tidb-operator.md (hunk 2 above)
Comment (DanielZhangQD) [reply zh/upgrade-tidb-operator.md:27]: @ran-huang 看下这样可以不

Comment (ran-huang) [reply zh/upgrade-tidb-operator.md:27]: “否则”指代不明，我改了一下，是这个意思吗

File: zh/upgrade-tidb-operator.md [hunk 5]
```diff
@@ -132,15 +152,25 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/upgrade-tidb-operator/']
 
 2. 将下载的文件和镜像上传到需要升级的服务器上，在服务器上按照以下步骤进行安装：
 
-    1. 安装 TiDB Operator 需要的 `crd.yaml` 文件：
+    1. 如果是第一次升级 Operator 到 v1.3.0 及以后版本，需要先创建新增加的 TidbNGMonitoring CRD。
```
Comment (DanielZhangQD) [inline zh/upgrade-tidb-operator.md:155]:
```suggestion
    1. 如果 TiDB Operator 从 v1.2.x 及更早版本升级到 v1.3.x 及以后版本，需要先执行下面命令创建新增加的 TidbNGMonitoring CRD。如果是 v1.3.x 及以后版本的 TiDB Operator 升级，可跳过这一步。
```

File: zh/upgrade-tidb-operator.md [hunk 6]
```diff
@@ -132,15 +152,25 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/upgrade-tidb-operator/']
...
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        kubectl create -f ./crd.yaml
+        ```
+
+        对于其他 CRD，执行该命令时会报 "AlreadyExists" 的错误，可以忽略该错误。
+
+    2. 安装 TiDB Operator 需要的 `crd.yaml` 文件：
```
Comment (DanielZhangQD) [inline zh/upgrade-tidb-operator.md:165]:
```suggestion
    2. 升级 TiDB Operator 需要的 `crd.yaml` 文件：
```

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Comment by KanShiori** [comment]
@ran-huang  PTAL

File: zh/upgrade-tidb-operator.md [hunk 7]
```diff
@@ -21,24 +21,44 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/upgrade-tidb-operator/']
     如果输出中未包含你需要的新版本，可以使用 `helm repo update` 命令更新 repo。详情请参考[配置 Helm repo](tidb-toolkit.md#配置-helm-repo)。
 
 2. 更新 Kubernetes 的 CustomResourceDefinition (CRD)。关于 CRD 的更多信息，请参阅 [CustomResourceDefinition](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/)。
-
+   
     * 如果 Kubernetes 版本大于等于 1.16:
 
-        {{< copyable "shell-regular" >}}
+        1. 如果 TiDB Operator 从 v1.2.x 及更早版本升级到 v1.3.x 及以后版本，需要先执行下面命令创建新增加的 TidbNGMonitoring CRD。如果是 v1.3.x 及以后版本的 TiDB Operator 升级，可跳过这一步。
```
Comment (ran-huang) [inline zh/upgrade-tidb-operator.md:27]: 前半句针对 v1.2.x 用户，需要 cherrypick 到 v1.2 文档吧

File: en/upgrade-tidb-operator.md [hunk 8]
```diff
@@ -23,22 +23,42 @@ If your server has access to the internet, you can perform online upgrade by tak
 2. Update [CustomResourceDefinition](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/) (CRD) for Kubernetes:
 
     * If your Kubernetes version >= v1.16:
+        
+        1. If upgrading Operator from v1.2.x or older to v1.3.x or later, you need to execute the following command to create the new TidbNGMonitoring CRD. If upgrading Operator from v1.3.x or later, you can skip this step.
```
Comment (ran-huang) [inline en/upgrade-tidb-operator.md:27]:
```suggestion
        1. If you upgrade TiDB Operator from v1.2.x or earlier versions to v1.3.x or later versions, you need to execute the following command to create the new TidbNGMonitoring CRD. If you upgrade TiDB Operator from v1.3.x or later versions, you can skip this step.
```

File: en/upgrade-tidb-operator.md [hunk 9]
```diff
@@ -132,15 +148,25 @@ If your server cannot access the Internet, you can offline upgrade by taking the
...
-    1. Install the `crd.yaml` file for TiDB Operator:
+    1. If upgrading Operator from v1.2.x or older to v1.3.x or later, you need to exec the following command to create the new TidbNGMonitoring CRD. If upgrading Operator from v1.3.x or later, you can skip this step.
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        kubectl create -f ./crd.yaml
+        ```
+
+        For other CRDs, there is a "AlreadyExists" error after executing this command, you can ingore this error.
```
Comment (ran-huang) [inline en/upgrade-tidb-operator.md:159]:
```suggestion
        After executing this command, you can expect to see an "AlreadyExists" error for other CRDs. You can ignore this error.
```

File: zh/upgrade-tidb-operator.md (hunk 7 above)
Comment (ran-huang) [reply zh/upgrade-tidb-operator.md:27]: 嗯，暂时不 cp 吧

File: en/upgrade-tidb-operator.md [hunk 10]
```diff
@@ -23,22 +23,42 @@ If your server has access to the internet, you can perform online upgrade by tak
...
+
+            ```bash
+            kubectl replace -f https://raw.githubusercontent.com/pingcap/tidb-operator/${operator_version}/manifests/crd.yaml && \
+            kubectl get crd tidbclusters.pingcap.com
+            ```
 
     * If your Kubernetes version < v1.16:
 
-        {{< copyable "shell-regular" >}}
+        1. If upgrading Operator from v1.2.x or older to v1.3.x or later, you need to execute the following command to create the new TidbNGMonitoring CRD. If upgrading Operator from v1.3.x or later, you can skip this step.
```
Comment (ran-huang) [inline en/upgrade-tidb-operator.md:46]:
```suggestion
        1. If you upgrade TiDB Operator from v1.2.x or earlier versions to v1.3.x or later versions, you need to execute the following command to create the new TidbNGMonitoring CRD. If you upgrade TiDB Operator from v1.3.x or later versions, you can skip this step.
```

File: en/upgrade-tidb-operator.md [hunk 11]
```diff
@@ -132,15 +148,25 @@ If your server cannot access the Internet, you can offline upgrade by taking the
 
 2. Upload the downloaded files and images to the server where TiDB Operator is deployed, and install the new TiDB Operator version:
 
-    1. Install the `crd.yaml` file for TiDB Operator:
+    1. If upgrading Operator from v1.2.x or older to v1.3.x or later, you need to exec the following command to create the new TidbNGMonitoring CRD. If upgrading Operator from v1.3.x or later, you can skip this step.
```
Comment (ran-huang) [inline en/upgrade-tidb-operator.md:151]:
```suggestion
    1. If you upgrade TiDB Operator from v1.2.x or earlier versions to v1.3.x or later versions, you need to execute the following command to create the new TidbNGMonitoring CRD. If you upgrade TiDB Operator from v1.3.x or later versions, you can skip this step.
```

**Review by ran-huang — APPROVED** [review]
Rest LGTM

**Review by ran-huang — APPROVED** [review]

**Pull request merged**